docs/architecture/graphql.rst

```diff
@@ -0,0 +1,109 @@
+GraphQL API (Beta)
+====================
+
+.. note::
+
+    The GraphQL API is in the early version and is subject to change.
+
+
+Saleor supports GraphQL API. It allows to create apps that can handle fulfillment, orders, pages, product and shipping management.
+
+GraphQL allows for more efficient data retrieval - it is easier to fetch all desired resources without redundant ones.
+
+Learn more at `official website <https://graphql.org>`_.
+
+Endpoint
+--------
+You can test API under ``/graphql`` endpoint - in default Saleor configuration URL would be ``http://localhost:8000/graphql/``. You can perform queries normally, however you need to log in to an account with proper permissions to run mutations and query for restricted data.
```

Comment:
API is available under ``/graphql`` endpoint. Requests must be sent using HTTP `POST` method and `application/json` content type.

With the `DEBUG=True` setting enabled, Saleor exposes an interactive GraphQL editor under ``/graphql``, that allows to access the API from the browser.